src/EnergyPlus/FanCoilUnits.hh

```diff
@@ -389,6 +390,10 @@ namespace FanCoilUnits {
                                   Array1D<Real64> const &Par // Function parameters
     );
 
+    Real64 CalcFanCoilHeatCoilPLRResidual(EnergyPlusData &state, Real64 const CyclingR,  // electric heating coil cycling ratio
+        Array1D<Real64> const &Par // Function parameters
+    );
+
```

Comment:
The new electric heating coil load residual calculation function for *Cycling Ratio*.